src/client/cl_predict.c

```diff
@@ -37,13 +37,12 @@ static int32_t Cl_HullForEntity(const entity_state_t *ent) {
 		return mod->head_node;
 	}
 
-	vec3_t emins, emaxs;
-	UnpackBounds(ent->bounds, emins, emaxs);
+	const cl_entity_t *ent = &cl.entities[s->number];
 
-	if (ent->client) {
-		return Cm_SetBoxHull(emins, emaxs, CONTENTS_MONSTER);
+	if (s->client) {
+		return Cm_SetBoxHull(ent->mins, ent->maxs, CONTENTS_MONSTER);
 	} else {
-		return Cm_SetBoxHull(emins, emaxs, CONTENTS_SOLID);
+		return Cm_SetBoxHull(ent->mins, ent->maxs, CONTENTS_SOLID);
 	}
```

Comment:
Entities now have their `mins` and `maxs` readily available, so there's no need to unpack them here anymore.